webcompat/static/css/development/components/issue-detail.css

```diff
@@ -172,6 +172,35 @@
   content: "\f00d";
 }
 
+/* QR code launcher */
+.Qr-wrapper {
```

Comment:
Sure, done. I copied the structure of the ```LabelEditor``` code so maybe that should be moved to a single file as well.